### ISSUE TITLE
Styled menu portals do not clear slots when changing between all and open games

### ISSUE THREAD

**Issue by haykam821** [NucleoidMC/nucleoid-extras#162]
<img width="578" alt="image" src="https://github.com/NucleoidMC/nucleoid-extras/assets/24855774/61148a43-6abc-4c51-9059-69dd0bb3f43b">

**Comment by ZekeZDev** [comment]
yep, looks like the issue I had the other day. Still happens in the latest version.
